templates/common/infra/bicep/core/database/cosmos/cosmos-account.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'To create a cosmos account.'
```

Comment:
Creates a Cosmos Account

Reply:
Descriptions has been updated following this example.